main/lsp/requests/completion.cc

```diff
@@ -734,6 +767,24 @@ vector<unique_ptr<CompletionItem>> allSimilarConstantItems(const core::GlobalSta
     return items;
 }
 
+string_view identPrefix(string_view name) {
```

Comment:
This was a dumb idea I eventually realized we didn't need; we can just check the prefix in `getCompletionItems` to determine whether we need to operate on local variables or on some kind of field.